### DISCUSSION TITLE
in quasar i setup it up but when i write at that time input formatting not working

### DISCUSSION THREAD

**Discussion by amit772000** [dm4t2/vue-currency-input#412] [Q&A]
quasar version : 2.6.0
vue : 3.0.0

here i use following setting as described in doc but when i type it is unable to format value instantly it format values only after blur 

useCurrencyInput({
        currency: props.currency,
        useGrouping: true,
        hideCurrencySymbolOnFocus: false,
        hideGroupingSeparatorOnFocus: false,
      });

**Comment by dm4t2** [comment]
There is a working Quasar example in the docs: https://dm4t2.github.io/vue-currency-input/examples.html#quasar

**Reply by amit772000** [reply]
okay i have worked now. it was vue version issue. i was using vue 3.0.0 now i use same version as you 3.2.29 and works. 

can you please suggest me i want to use debounce to update model value after specific number of seconds . can you please suggest solution. i use debounce in q-input but not working.
